lib/utils.js

```diff
@@ -1,8 +1,27 @@
-export function PDFNumber(n) {
+const fArray = new Float32Array(1);
+const uArray = new Uint32Array(fArray.buffer);
+
+export function PDFNumber(n, roundUp = false) {
```

Comment:
Is this function called with roundUp = true?

Reply:
by default it will roundDown

Reply:
I think we could simplify and only implement roundDown. If needed later implement roundUp

Reply:
only reason i left both was that if we want to do `float64 > float32` it would need to be rounded up